applications/ConstitutiveLawsApplication/custom_constitutive/generic_small_strain_isotropic_plasticity.cpp

```diff
@@ -105,7 +105,7 @@ void GenericSmallStrainIsotropicPlasticity<TConstLawIntegratorType>::CalculateMa
     } else { // We check for plasticity
         // Integrate Stress plasticity
         Vector& r_integrated_stress_vector = rValues.GetStressVector();
-        const double characteristic_length = ConstitutiveLawUtilities<VoigtSize>::CalculateCharacteristicLength(rValues.GetElementGeometry());
+        const double characteristic_length = AdvancedConstitutiveLawUtilities<VoigtSize>::CalculateCharacteristicLength(rValues.GetElementGeometry());

```

Comment:
Question, did we solved the issue of the characteristic length in the initial configuration?

Reply:
@AlejandroCornejo ping, did we solved the issue?

Reply:
Now we have the two methods... You can compute the char lenght in the original or updated conf